src/Kanbas/Courses/Quizzes/List/index.tsx

```diff
@@ -1,22 +1,145 @@
+  ) => {
+    const today = new Date();
+
+    if (today < availableFromDate) {
+      return (
         <>
-            {/* TODO - update this to pass the quiz id to this function for navigation */}
-            <a onClick={() => handleQuizSelection()} />
+          <b>Not Available until</b> {availableFromDate.toDateString()}{" "}
         </>
-    );
+      );
+    }
+
+    if (today < availableUntilDate) {
+      return <b>Available</b>;
+    }
+
+    return <b>Closed</b>;
+  };
+
+  useEffect(() => {
+    if (courseId !== undefined) {
+      getCourseQuizzes(courseId).then((quizzes: IKanbasQuiz[]) => {
+        dispatch(setQuizzes(quizzes));
+      });
+    }
+  }, [dispatch]);
+
+  return (
+    <>
+      <ul className="list-group wd-quizzes-grid">
+        <li className="list-group-item">
+          <div
+            className="wd-quizzes-grid-section d-flex align-items-center"
+            onClick={() => setQuizzesExpanded(!quizzesExpanded)}
+            aria-controls={"quiz-section-collapse"}
+            aria-expanded={quizzesExpanded}
+          >
+            <FaCaretDown
+              style={{ transform: quizzesExpanded ? "" : "rotate(-90deg)" }}
+            />
+            <div className="ms-1 me-auto">Assignment Quizzes</div>
+          </div>
+
+          <Collapse in={quizzesExpanded}>
+            <div
+              id={"quiz-section-collapse"}
+              className="wd-quiz-section-collapse"
+            >
+              <ul className="list-group">
+                {quizzesList?.map((q, qIndex) => (
+                  <li
+                    key={qIndex}
+                    className="list-group-item d-flex align-items-center"
+                  >
+                    <FaRocket className="ms-2 wd-icon-green" />
+                    <div className="wd-quizzes-grid-content-text flex-fill me-auto">
+                      <a
+                        onClick={() => handleQuizSelection(q._id)}
+                        className="wd-event-link"
+                      >
+                        {q.title}
+                      </a>
+                      <p className="wd-quizzes-grid-content-details">
+                        {getAvailableDateDisplay(
+                          q.availableDate,
+                          q.availableUntilDate
+                        )}{" "}
+                        | Due {q.dueDate.toString()} | {q.points} Points |{" "}
+                        {q.questionsCount} Questions
+                      </p>
+                    </div>
+                    <span className="float-end wd-quizzes-grid-content-actions d-flex align-items-center">
+                      {q.isPublished && (
+                        <FaCheckCircle
+                          onClick={() => handlePublishToggle(q._id)}
+                          className="me-1 wd-icon-green"
+                        />
+                      )}
+                      {!q.isPublished && (
+                        <FaBan
```

Comment:
Shouldn't we apply `text-muted` bootstrap for closed ones?

Reply:
The color is muted for the published icon, not the text. I will add this change.